pom.xml

```diff
@@ -471,13 +471,6 @@
         <groupId>com.clearspring.analytics</groupId>
         <artifactId>stream</artifactId>
         <version>2.7.0</version>
-        <exclusions>
-          <!-- Only HyperLogLogPlus is used, which doesn't depend on fastutil -->
-          <exclusion>
-            <groupId>it.unimi.dsi</groupId>
-            <artifactId>fastutil</artifactId>
```

Comment:
I'm very open to suggestions.

Note that parquet is already bringing in part of it via it's own assembly jar renamed under parquet/.

I haven't worked with sbt-assembly rename, but is that a solution?

Perhaps we should all use a different implementation other than QDigest like tdigest?

BTW, I'm unclear where discussion is best placed.  Should it be here or there: https://issues.apache.org/jira/browse/SPARK-6029

The more I look into this sort of ```dependency **ll```, the more I believe it is intractable in the current way we (generally, not Spark specifically) package things.